docs/source/docformat/restructuredtext.rst

```diff
@@ -48,27 +48,27 @@ Note that *docutils* requires a space before and after the ``:`` used to mark cl
 .. code:: python
 
     def fox_speed(size, weight, age):
-        """
+        '''
         :Parameters:
             size
                 The size of the fox (in meters)
             weight : float
                 The weight of the fox (in stones)
             age : int
                 The age of the fox (in years)
-        """
+        '''
```

Comment:
PEP 257 recommends using `"""`, so I'd prefer to use that in examples as well. Was there a particular reason for this change? I couldn't find any in the commit comment of 9c2898e8.

Reply:
I honestly don't know !